python/graphstorm/sagemaker/sagemaker_train.py

```diff
@@ -213,6 +224,11 @@ def run_train(args, unknownargs):
         data_path, sagemaker_session)
     graph_config_path = download_graph(graph_data_s3, graph_name,
         host_rank, data_path, sagemaker_session)
+    if model_checkpoint_s3 is not None:
+        # Download Saved model checkpoint to resume
+        download_model(model_checkpoint_s3, restore_model_path, sagemaker_session)
+        print(f"{restore_model_path} {os.listdir(restore_model_path)}")
```

Comment:
Let's use logging instead of printing. `INFO` level would be fine here.

Reply:
Let's do it in another PR.

Reply:
We need to update all the scripts under sagemaker folder.